repositories/SectionRepository.js

```diff
@@ -99,3 +120,52 @@ module.exports.duplicateSection = function duplicateSection(id, position) {
     });
   });
 };
+
+module.exports.createIntroduction = function createIntroduction(
```

Comment:
Do we need to create these CRUD methods? Can we not leverage the `update` method?